Source/Resources/Content/MSGraph/DeviceManagement/DeviceEnrollmentConfigurations/deviceEnrollmentPlatformRestrictionsConfiguration - Device Enrollers can enroll personal devices.json

```diff
@@ -18,7 +18,7 @@
       }
     }
   ],
-  "description": "Allows a device enroller to enroll any Windows device, including personal devices. A device without an autopilot profile is considered personal until it is enrolled.",
+  "description": "What is it: Allows a device enroller to enroll any Windows device, devices without an autopilot profile. \n Simeon Notes: This policy allows you to specify users that have the ability to enroll devices into your environment. Applies to users in the Azure AD group 'Builtin – Device Enrollers'. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/enrollment/enrollment-restrictions-set",
```

Comment:
**What does this do?**
Allows you to specify users that have the ability to enroll devices into Intune. The baseline allows users in the Azure AD group 'Baseline – Device Enrollers' to enroll any Windows device, and allows all other users only to enroll devices that have been previously registered with Autopilot. This limit can be overridden by other configurations with a higher priority.

**Why should you use this?**
If you want to restrict general users from enrolling personal Windows devices without autopilot.

**What is the end user impact?**
Only certain users will be able to enroll devices.

**Learn more**
[Set enrollment restrictions](https://docs.microsoft.com/en-us/mem/intune/enrollment/enrollment-restrictions-set)